WebRoot/js/ajax/dwt/core/Dwt.js

```diff
@@ -556,7 +556,13 @@ function(htmlElement, point) {
 Dwt.setLocation =
 function(htmlElement, x, y) {
 	if (!(htmlElement = Dwt.getElement(htmlElement))) { return; }
-	var position = DwtCssStyle.getProperty(htmlElement, 'position');
+	var position;
```

Comment:
I think we can simply do :

```
var position = DwtCssStyle.getProperty(htmlElement, 'position');
if (!position) {
position = htmlElement.style.position;
}
```

Reply:
I think it works. From testing perspective, however, we should search all functions which call Dwt.setLocation and execute regression tests (i.e. confirm not only loading process but also all functions/features which call Dwt.setLocation work in the same way as before) on all supported browsers if we apply the code in your comment. What do you think?

Reply:
I think it works. From testing perspective, however, we should execute regression tests not only on Edge but also on all supported browsers if we apply the code in your comment. The current change affects only on Edge. What do you think?

Reply:
I am agree.
But what if some other browser or environment will lend us again with this situation? Its better to make generalised fix instead of remain browser-dependent.